osf_tests/test_collection_submission.py

```diff
@@ -0,0 +1,230 @@
+import pytest
+
+from osf_tests.factories import (
+    AuthUserFactory,
+)
+
+from transitions import MachineError
+
+from osf_tests.factories import NodeFactory, CollectionFactory, CollectionProviderFactory
+
+from osf.models import CollectionSubmission
+from osf.utils.workflows import CollectionSubmissionStates
+from framework.exceptions import PermissionsError
+from api_tests.utils import UserRoles
+
+POST_URL = '/v2/collection_submissions_actions/'
+
```

Comment:
Can you add some tests for email rendering? See https://github.com/CenterForOpenScience/osf.io/blob/develop/osf_tests/test_schema_responses.py#L715 as an example of how we do this with other moderation emails